README.md

```diff
@@ -1,21 +1,26 @@
-# gateway-certs-generator
+# Certificate Generator for Slice Gateway
 
-This is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to easy-rsa (typically bundled with OpenVPN).
+The `gateway-certs-generator` is an opinionated single-file OpenVPN TLS certificate configuration generator for slice gateways. it is an enhancement to `easy-rsa` (typically bundled with OpenVPN).
 
-easy-rsa is a CLI utility to build and manage a PKI CA. In layman's terms, this means to create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
+To create and maintain a PKI CA, use the CLI tool `easy-rsa`. Using the tool create a root certificate authority, and request and sign certificates including intermediate CAs and certificate revocation lists (CRL).
 
 ## Getting Started
-It is strongly recommended to use a released version. Follow the instructions in this [document](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
+It is strongly recommended to use a released version. 
 
-## Building and Deploying `gateway-certs-generator` and use with `kubeslice-controller` in a Local Kind Cluster
-For more information, see [getting started with kind clusters](https://docs.avesha.io/documentation/open-source/0.2.0/getting-started-with-kind-clusters).
+Please refer to our documentation on:
+- [Installing KubeSlice on cloud clusters](https://kubeslice.io/documentation/open-source/0.5.0/getting-started-with-cloud-clusters/installing-kubeslice/installing-the-kubeslice-controller).
+- [Installing KubeSlice on kind clusters](https://kubeslice.io/documentation/open-source/0.5.0/tutorials/kind-install-kubeslice-controller).
 
-### Prerequisites
+## Building and Deploying `gateway-certs-generator` on a Kind Cluster
+
+To create certificates, the controller needs the `gateway-cert-generator` image. Therefore, we create the image and use the image version in the controller values file.
 
-* Docker installed and running in your local machine
-* A running [`kind`](https://kind.sigs.k8s.io/)  cluster
+### Prerequisites
+Before you begin, make sure the following prerequisites are met:
+* Docker is installed and running on your local machine.
+* A running [`kind`](https://kind.sigs.k8s.io/) cluster.
 * [`kubectl`](https://kubernetes.io/docs/tasks/tools/) installed and configured
```

Comment:
kubectl is installed and configured ( add a period at the end of the sentence) - as other items are full sentences.

Reply:
done